examples/workflows/run_ensemble_model_noise.py

```diff
@@ -18,6 +18,14 @@
 import logging
 from functools import partial
 import torch
+import os
+
+# Set number of GPUs to use to 1
+os.environ["WORLD_SIZE"] = "1"
+# Set model registry as a local folder
+model_registry = os.path.join(os.path.dirname(os.path.realpath(os.getcwd())), "models")
+os.makedirs(model_registry, exist_ok=True)
+os.environ["MODEL_REGISTRY"] = model_registry
```

Comment:
Can we explore the more general fix to this? See https://github.com/NVIDIA/earth2mip/issues/61. Should be a 1 liner in earth2mip/__init__.py.

Reply:
I updated the default in the _config.